designs/2019-config-tester/README.md

```diff
@@ -0,0 +1,108 @@
+- Start Date: 2019-06-14
+- RFC PR: https://github.com/eslint/rfcs/pull/27
+- Authors: Toru Nagashima &lt;https://github.com/mysticatea&gt;
+
+# Config Tester
+
+## Summary
+
+Providing `ConfigTester` API that tests shareable configs and plugin's preset configs.
+
+## Motivation
+
+Currently, we don't provide stuff to test shareable configs. This means the community doesn't have an easy way to check if their config is good or not.
+
+- Is the config structure valid?
+- Does every rule exist?
+- Does every rule have valid options?
```

Comment:
see above for changes across eslint version

Reply:
I think that is not false positive because ESLint itself does the same validation (non-existence rules are ignored).